.github/workflows/edfi_paging_test.yml

```diff
@@ -52,11 +52,23 @@ jobs:
           key: ${{ runner.os }}-poetry-${{ hashFiles(env.LOCK_FILE) }}
           restore-keys: |
             ${{ runner.os }}-poetry-
+
       - name: Run CI Test Task
         run: python ./eng/build.py ci:test ${{ env.PACKAGE_NAME }}
 
-      - name: Archive code coverage results
-        uses: actions/upload-artifact@e448a9b857ee2131e752b06002bf0e093c65e571
+      - name: Test Report
+        uses: dorny/test-reporter@0d00bb14cb0cc2c9b8985df6e81dd333188224e1 #v1.5.0
+        if: success() || failure()
+        with:
+            name: Pytest results
+            path: "**/pytest-junit.xml"
+            reporter: java-junit
+
+      - name: Test Report
+        uses: dorny/test-reporter@0d00bb14cb0cc2c9b8985df6e81dd333188224e1 #v1.5.0
+        if: success() || failure()
         with:
-          name: code-coverage-report
-          path: ${{ env.SRC_DIR }}/htmlcov/
\ No newline at end of file
+            name: MyPy results
+            path: "**/mypy-junit.xml"
+            reporter: java-junit
```

Comment:
These two new actions will capture the unit test and static type test output directly in GitHub Actions.